Formula/sigrok-cli.rb

```diff
@@ -0,0 +1,38 @@
+class SigrokCli < Formula
+  desc "Sigrok command-line interface to use logic analyzers and more"
+  homepage "https://sigrok.org/"
+  license "GPL-3.0-or-later"
+
+  head "git://sigrok.org/sigrok-cli", branch: "master"
+
+  stable do
+    url "git://sigrok.org/sigrok-cli",
+        tag:      "sigrok-cli-0.7.2",
+        revision: "b584f959edb788f1731d5a304badf241ac21bf65"
+  end
```

Comment:
```suggestion
  url "git://sigrok.org/sigrok-cli",
      tag:      "sigrok-cli-0.7.2",
      revision: "b584f959edb788f1731d5a304badf241ac21bf65"
  license "GPL-3.0-or-later"
  head "git://sigrok.org/sigrok-cli", branch: "master"
```
Do we really need a Git clone? Is there no source tarball?

Reply:
Fixed